common/web/keyboard-processor/src/keyboards/defaultLayouts.ts

```diff
@@ -275,9 +279,6 @@ export class Layouts {
             case "K_SHIFT":
               shiftKey=key;
               break;
-            case "K_TAB":
-              nextKey=key;
-              break;
```

Comment:
Is this a lurking bug -- we aren't handling <kbd>Tab</kbd> in the way we thought we were?

Reply:
My understanding is that we track the other keys because key-states can motivate us to highlight them on the active layer.  As for shift, it has other special handling attached.

In regard to TAB, I _believe_ the only real reason to look for it, in particular, is to facilitate a reverse-shift image on the shift layer to match standard shift-tab behavior.  That's handled in other ways.